src/main/scala/no/ndla/draftapi/service/ConverterService.scala

```diff
@@ -445,6 +445,11 @@ trait ConverterService {
           Failure(new ValidationException(errors = Seq(error)))
         case None => Success(partiallyConverted)
         case Some(lang) =>
+          val mergedVisualElement =
```

Comment:
Det bør vel egentlig legges til validering på dette også. Hvis ikke risikerer vi at data brukeren potensielt sender inn blir fjernet uten at det blir gitt noen form for beskjed

Reply:
Et problem med det er at for allerede importerte artikler der vi har visualelement så er det ingen måte å redigere disse i editor. Dei skulle ikkje vore der i utgangspunktet.

Reply:
Da bør vel enten de det gjelder re-importeres, eller hvis det ikke lar seg gjøre, kanskje migreres?

Reply:
Denne koden var et forsøk på å håndtere reimport som ikkje fungerte som det skulle fordi patching av artikkel ikkje sletta visualelement. Det hadde kanskje vore bedre med en migrering.